### ISSUE TITLE
Have another look at BEM

### ISSUE THREAD

**Issue by rymatech** [fac-17/IJKL-week2-project#28]
Bem can be a pain at first and you do not need to adopt all of its teaching but perhaps consider only using classes in the CSS. So no change tag elements or using IDs

**Comment by rooberrydev** [comment]
yeah definitely, on our first project we stuck to just using classnames. in this one, we had skeleton html with ids declared before we added anything so we just used those in the css. I agree that using classes only would be much better. will attempt to do this in the refactor.
